app/licenses.yml

```diff
@@ -132,3 +132,11 @@
   license: The Apache Software License, Version 2.0
   licenseUrl: http://www.apache.org/licenses/LICENSE-2.0.txt
   url: https://github.com/chrisjenx/Calligraphy
+- artifact: org.jetbrains.kotlin:kotlin-runtime:+
+  name: kotlin-runtime
+  copyrightHolder: JetBrains Inc.
+  license: The Apache Software License, Version 2.0
+- artifact: org.jetbrains.kotlin:kotlin-stdlib:+
+  name: kotlin-stdlib
+  copyrightHolder: JetBrains Inc.
+  license: The Apache Software License, Version 2.0
```

Comment:
You should check the "ensure line feed at file end on Save" in Android Studio 😎 

<img width="569" alt="screen shot 2017-02-07 at 9 17 44" src="https://cloud.githubusercontent.com/assets/101800/22672314/62c0b8c4-ed16-11e6-8a8a-ae210ef07b8b.png">